src/vtkIECTransformLogic.h

```diff
@@ -46,7 +46,7 @@ class vtkGeneralTransform;
 /// RT beam node to get the geometrical parameters defining the state of the objects involved.
 /// Image describing these coordinate frames:
 /// https://github.com/SlicerRt/SlicerRtDoc/blob/master/technical/IEC%2061217-2002_CoordinateSystemsDiagram_HiRes.png
-///
+/// The RAS coordinate system is not part of IEC but we leave it here as a helper function since it's used e.g. in Slicer https://slicer.readthedocs.io/en/latest/user_guide/coordinate_systems.html
```

Comment:
```suggestion
/// \note The RAS coordinate system is not part of IEC but we leave it here as a helper function since it's used e.g. in Slicer https://slicer.readthedocs.io/en/latest/user_guide/coordinate_systems.html
/// Same applies to DICOM LPS coordinate system
```